examples/mne_util.py

```diff
@@ -0,0 +1,294 @@
+        # brain.add_label from pysurfer?
+        if isinstance(label, mne.Label):
+            hi = 0 if label.hemi == 'lh' else 1
+            label_vert_idx = self.get_label_vinds(label)
+            varray = src[hi]['vertno'][label_vert_idx]
+        elif isinstance(label, mne.BiHemiLabel):
+            lh_label_vert_idx, rh_label_vert_idx = self.get_label_vinds(label)
+            varray = [src[0]['vertno'][lh_label_vert_idx],
+                      src[1]['vertno'][rh_label_vert_idx]]
+        return varray
+
+    def get_hemi_idx(self, label):
+        if isinstance(label, mne.Label):
+            return 0 if label.hemi == 'lh' else 1
+        elif isinstance(label, mne.BiHemiLabel):
+            hemis = [None] * 2
+            for i, lab in enumerate([label.lh, label.rh]):
+                hemis[i] = 0 if lab.hemi == 'lh' else 1
+            return hemis
+
+
+def morph_labels(labels, subject_to, subjects_dir=None):
+    """ morph labels from fsaverage to specified subject """
+
+    if subjects_dir is None:
+        subjects_dir = mne.utils.get_subjects_dir()
+
+    if isinstance(labels, mne.Label):
+        labels = [labels]
+
+    labels_morphed = list()
+    for lab in labels:
+        if isinstance(lab, mne.Label):
+            labels_morphed.append(lab.copy())
+        elif isinstance(lab, mne.BiHemiLabel):
+            labels_morphed.append(lab.lh.copy() + lab.rh.copy())
+
+    for i, l in enumerate(labels_morphed):
+        if l.subject == subject_to:
+            continue
+        elif l.subject == 'unknown':
+            print("uknown subject for label %s" % l.name,
+                  "assuming if is 'fsaverage' and morphing")
+            l.subject = 'fsaverage'
+
+        if isinstance(l, mne.Label):
+            l.values.fill(1.0)
+            labels_morphed[i] = l.morph(subject_to=subject_to,
+                                        subjects_dir=subjects_dir)
+        elif isinstance(l, mne.BiHemiLabel):
+            l.lh.values.fill(1.0)
+            l.rh.values.fill(1.0)
+            labels_morphed[i].lh = l.lh.morph(subject_to=subject_to,
+                                              subjects_dir=subjects_dir)
+            labels_morphed[i].rh = l.rh.morph(subject_to=subject_to,
+                                              subjects_dir=subjects_dir)
+
+    # make sure there are no duplicate labels
+    labels_morphed = sorted(list(set(labels_morphed)), key=lambda x: x.name)
+
+    return labels_morphed
+
+
+def apply_projs(epochs, fwd, cov):
+    """ apply projection operators to fwd and cov """
+    proj, _ = mne.io.proj.setup_proj(epochs.info, activate=False)
+    G = fwd['sol']['data']
+    fwd['sol']['data'] = np.dot(proj, G)
+
+    Q = cov.data
+    if not np.allclose(np.dot(proj, Q), Q):
+        Q = np.dot(proj, np.dot(Q, proj.T))
+        cov.data = Q
+
+    return fwd, cov
+
+
+def scale_sensor_data(epochs, fwd, cov, roi_to_src, eeg_scale=1., mag_scale=1.,
```

Comment:
We probably don't really need this function. Or at least it doesn't need to be public. In other words, it should have an underscore prefix `_` prepended to the name to make it clear that it's not intended for users to ever see or use it.